homeassistant/components/generic/camera.py

```diff
@@ -172,6 +173,12 @@ def __init__(
         self._last_url = None
         self._last_image = None
 
+        self._attr_device_info = DeviceInfo(
+            entry_type=DeviceEntryType.SERVICE,
```

Comment:
This is mostly mean to connect devices, not services. This last addition doesn't make much sense to me.

Even connected through the internet, these are still cameras.

```suggestion
```

If this was done because of providing images, then the answer is: This domain should no longer be used for that (we now have an image entity for those).

../Frenck

Reply:
Oh yes, I didn't have the "image" entity in mind anymore, that makes perfect sense then. Will revert this.